src/components/LoanCards/Buttons/LendButton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+	<kv-button
+		class="button"
+		v-kv-track-event="['Lending', 'click-Read more', 'lend-button-click', id, 'true']"
+		v-on="listeners">
+		Lend $25
+	</kv-button>
+</template>
+
+<script>
+import KvButton from '@/components/Kv/KvButton';
+
+export default {
+	components: {
+		KvButton,
+	},
+	props: {
+		id: {
+			type: Number,
+			default: null
+		},
+		listeners: {
+			type: Number,
+			default: null
```

Comment:
When I didn't have listeners defined in props it was giving me a console error, however I'm not sure I've actually defined this correctly, @mcstover thoughts?

Reply:
It's `$listeners`. All the vue internal variables/functions start with a dollar sign. So you should change the usage in the template to `v-on="$listeners"` and remove the extra prop.